### ISSUE TITLE
Fish Voucher Collection Method

### ISSUE THREAD

**Issue by kyule** [BioKIC/NEON-Biorepository-Data-Docs#17]
Via Laura S. in 11/5/19 meeting

The sampling method is listed on the specimen labels (electrofisher, minnow trap, etc) but is not available in the portal. Is there a way to extract this from the API to add to the portal?

Via KY:

I believe these methods are consistent within site (Laura S. can confirm). Therefore, maybe a table that converts site to method and places method in a field in the occurrence record could work?

For several collections, collection method needs to be included in the sample details as it can vary quite a bit within the collection.

**Comment by arbolitoloco** [comment]
Per NEON's API:

`https://data.neonscience.org/api/v0/samples/view?sampleTag=FSH.MCRA.02.20181001.1.0012&sampleClass=fsh_perFish_in.voucherSampleID`

returns only shipment date, locations, dna info, and other IDs.

This is a current limitation of NEON's API. We don't currently know what information is available from the API for each sample class. 

Until the API is further developed, to display all the information relevant to Symbiota, the only way to fix this is to upload the data gathered from NEON's internal spreadsheets (if that information resides there).

**Comment by kyule** [comment]
Right. I thought the same method was always used within a site, making it easy to know the method without it being exposed on the API. That is actually incorrect for the 4 lake sites, so nevermind. The method is written on the bottle so if it never gets exposed this is the kind of thing a student could input given relatively low no/year of fish vouchers.

**Comment by kyule** [comment]
Will import from R if someone expresses interest. Otherwise, wait until API improvements occur.

**Comment by kyule** [comment]
Kelsey will open  ServiceNow
